docs/sources/mimir/release-notes/v2.13.md

```diff
@@ -0,0 +1,90 @@
+
+- **Mimirtool can now verify the validity of a Mimir runtime configuration** file with the `mimirtool runtime-config verify` command.
+
+- **Configuration of TLS for S3 buckets** is now possible via `-<prefix>.s3.http.*` and `-common.storage.s3.http.*` configuration options.
+
+- **Improved OTLP ingestion performance** by using native translation, which reduces memory usage in the distributor by up to 30% and CPU usage by up to 8%.
+
+- **Active series are now updated along with owned series.** This means the number of active series for a tenant is much more accurate after adding ingesters.
+
+- **Remote read (`/prometheus/api/v1/read`) is becoming a first-class endpoint in Mimir** with support in `query stats` logs and query blocking in the query-frontend. More coming in future releases.
+
+- **Improved CPU performance in processing queries with regular expressions** that contain many alternations (for example `foo|bar|baz|...`).
+
+Additionally, the following previously experimental features are now considered stable:
+
+- Rules tenant federation via the `source_tenants` field in rule groups.
+- Enabling and recording and alerting rules evaluation on a per-tenant basis.
+- Limiting the number of tenants a federated query can query.
+
+## Important changes
+
+In Grafana Mimir 2.13 the following behavior has changed:
+
+- The default Docker image `grafana/mimir` is now based on the distroless image `gcr.io/distroless/static-debian12`.
+  See [Debugging distroless container images](https://grafana.com/docs/mimir/next/manage/mimir-runbooks/#debugging-distroless-container-images-in-kubernetes) for more details on how to
+  work with distroless images.
+
+- Errors logs in the ingester are now sampled at 10% by default. Sampled log lines will contain `(sampled 1/10)`. All the discarded samples will still be tracked by the `cortex_discarded_samples_total` metric.
+
+- Anonymous usage statistics now include actual CPU usage instead of available CPU cores.
+
+- Continuous-test is no longer a standalone binary and is now part of Mimir as its own target. The published Docker images has been updated to use the new packaging.
+
+The following deprecated configuration options are removed in Grafana Mimir 2.13:
+
+- The configuration option `-log.buffered`, which was deprecated in 2.11 and is now enabled by default.
+
+## Experimental features
+
+Grafana Mimir 2.13 includes new features that are considered experimental and disabled by default.
+Use them with caution and report any issues you encounter:
+
+- **New PromQL query engine** can be configured via the `-querier.promql-engine=mimir` CLI flag on queriers.
+  It aims to provide better utilization than the upstream Prometheus PromQL engine when processing large sets of data.
+
+- Support for [**HAProxy PROXY protocol**](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) can now be enabled via `-server.proxy-protocol-enabled=true`.
```

Comment:
Can we add some context for this? I have no idea what this means.

Reply:
My understanding is that PROXY protocol is a way to communicate information about client at the beginning of TCP connection. Our HTTP and gRPC servers support this to know who they communicate with, basically replacing information from TCP header.

Reply:
at a second look I don't think we should even be including this in the release notes. Mimir doesn't do anything with the client's address, so enabling this doesn't bring anything for Mimir. I found these docs in dskit which were added along with the feature

https://github.com/grafana/dskit/blob/8569e93a110e8b5642be030dbebccab5f6a41bad/server/PROXYPROTOCOL.md

If you agree, I'll just remove this line from the notes

Reply:
Yeah, I think that makes sense. This doesn't seem relevant for Mimir.

Reply:
Agree, don't see much benefit for Mimir users.

Reply:
```suggestion
```

> Unless they run Mimir directly behind some proxy that supports this protocol.

that looks niche. We still have the changelog entry